spec/unit/pops/pn_spec.rb

```diff
@@ -0,0 +1,95 @@
+#! /usr/bin/env ruby
+require 'spec_helper'
+require 'puppet/pops'
+require 'puppet/pops/pn'
+
+module Puppet::Pops::PN
+
+describe 'Puppet::Pops::PN' do
+  context 'Literal' do
+    context 'prodces a string with' do
+      it 'boolean' do
+        expect(Literal.new(true).to_s).to eql('true')
+      end
```

Comment:
The example text sequence is not really readable. "Puppet::Pops::PN Literal produces a string with boolean" - eh - as opposed to what, when... I have to read the logic to understand what it is an example of.

"has a to_s method that formats"... "boolean true as 'true'"